Xamarin.Forms.Core/Frame.cs

```diff
@@ -56,6 +58,29 @@ public float CornerRadius
 
 		int IBorderElement.CornerRadius => (int)CornerRadius;
 
+		void IFrameController.SetContentPadding(Thickness value)
```

Comment:
This is mainly for Android which has additional padding due to the border.

Reply:
I have a very bad feeling about this, but it's kinda private API, so let's go with it if we can't infer that information from the Measuring API

Reply:
I believe this is temporary... The AndroidX controls are not using this method anymore, so things will be grand! I hope...